Source/common/SNTConfigurator.m

```diff
@@ -101,6 +101,10 @@ @implementation SNTConfigurator
 static NSString *const kBlockedPathRegexKey = @"BlockedPathRegex";
 static NSString *const kBlockedPathRegexKeyDeprecated = @"BlacklistRegex";
 
+// TODO(markowsky): move these to sync server only.
+static NSString *const kMetricsFormat = @"ExportMetricsFormat";
```

Comment:
Add a comment listing the valid format types too

Reply:
Sorry, meant for valid values. I'm assuming it's something like "json" vs "proto"?

Reply:
I was going to put list the values in a future PR. 

Sketched out I have the following in my notes:

- **rawJson**: a full blob of the metrics (basically the raw metrics
- **json**: metrics rewritten to one metric per line for our system.
- **prometheus**:  serialized to the prometheus format.

I'll update this.

Reply:
@russellhancox I fixed the issue. Apparently I didn't push the last commit before I'd written the PR description.